pytorch_lightning/loops/fit_loop.py

```diff
@@ -215,13 +216,11 @@ def on_advance_start(self) -> None:
         self.trainer.accumulation_scheduler.on_train_epoch_start(self.trainer, self.trainer.lightning_module)
 
         # stores accumulated grad fractions per batch
-        self.epoch_loop.batch_loop.accumulated_loss = TensorRunningAccum(
-            window_length=self.trainer.accumulate_grad_batches
-        )
+        self.epoch_loop.batch_loop.accumulated_loss.window_length = self.trainer.accumulate_grad_batches
```

Comment:
This should reset the `TensorRunningAccum`. Either keep as before or add the reset() call.